include/cuco/static_map.cuh

```diff
@@ -91,7 +91,7 @@ template <class Key,
           cuda::thread_scope Scope = cuda::thread_scope_device,
           class KeyEqual           = thrust::equal_to<Key>,
           class ProbingScheme =
-            cuco::experimental::double_hashing<4,  // CG size
+            cuco::experimental::linear_probing<4,  // CG size
```

Comment:
I realized `double_hashing` is not the best option for a single map with a 50% load factor (which is the most common use case of a single map) thus changing this back to the default parameter (linear probing with CG size = 4) used by the legacy map

Reply:
Sounds good. Although I'm still not conviced we should use a default load factor as low as 50%. Hash maps shine because of their low memory overhead. I know there's a perf tradeoff and cudf has settled for 50%, but I tend to use 80% as my internal default.

Reply:
I think we should start thinking about how to implement automated benchmarking and visualization for two reasons: